pynopegl-utils/pynopegl_utils/viewer/__init__.py

```diff
@@ -165,7 +165,8 @@ def _select_script(self):
         script = QUrl(script).path()  # handle the file:// automatically added by Qt/QML
 
         if script.endswith(".py"):
-            dirname = "file://" + Path(script).resolve().parent.as_posix()
+            dirname = Path(script).resolve().parent.as_posix()
```

Comment:
Don't we want to always `resolve()` the path script so that it's always transformed and stored as an absolute path? (Here this is just done for the dirname, not the stored script value)

Reply:
Done.